styles.css

```diff
@@ -0,0 +1,24 @@
+.autocomplete {
+    position: absolute;
+    top: 74px;
+    width: 300px;
+    font-size: 15px;
+    color: #6d6a6a;
+    font-family: 'Fira Code', monospace;
```

Comment:
Should probably remove the reference to Fira Code; I used that in the sample application I made, but it depends on the font being loaded.